x-pack/plugins/gis/server/maps_telemetry/maps_usage_collector.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import { TASK_ID, scheduleTask, registerMapsTelemetryTask } from './telemetry_task';
+
+export function initTelemetryCollection(server) {
+  const { taskManager } = server;
```

Comment:
fluffy line. can just use `server.taskManager` like you do elsewhere in the file.